cmux-tui/bindings/python/tests/test_resource_api.py

```
@@ -1268,19 +1285,35 @@ def test_terminal_snapshot_accepts_protocol_one_tab_id_only(self) -> None:
+                "id": str(TERMINAL),
+                "tab_id": str(TAB),
+                "tab_ids": [str(TAB)],
+                "title": "dual",
+                "cols": 80,
+                "rows": 24,
+                "running": True,
+                "lifecycle": "running",
+            },
         ]
+        expected = [(TAB,), (), (TAB,)]
+        for response, tab_ids in zip(responses, expected):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)